set.mm

```diff
@@ -85113,6 +85138,76 @@ the first case of his notation (simple exponentiation) and subscript it
     ( cmap co wcel wf cvv wa wb elmapex elmapg syl ibi ) ABCDEFZCBAGZOBHFCHFIOP
     JABCKBCAHHLMN $.
 
+  ${
+    $d A f m $.  $d B f m $.  $d V m $.
+    $( The set exponentiation ` ( B ^m A ) ` is the set of all functions that
+       map from ` A ` to ` B ` .  Generalisation of ~ mapvalg for arbitrary
+       classes as domains instead of sets only.  (Contributed by AV,
+       8-Aug-2024.) $)
+    mapfset $p |- ( B e. V -> { f | f : A --> B } = ( B ^m A ) ) $=
+      ( vm wcel cv wf cab cmap co vex feq1 elab cvv simpr fdmexb mpbiri adantr
+      wa elmapd exbiri pm2.43b elmapi impbid1 syl5bb eqrdv ) BDFZEABCGZHZCIZBAJ
+      KZEGZUKFABUMHZUHUMULFZUJUNCUMELZABUIUMMNUHUNUOUHUNUOUNUHUOUNUNUHTBAUMDOUN
+      UHPUNAOFZUHUNUQUMOFUPABUMQRSUAUBUCUMBAUDUEUFUG $.
+  $}
+
+  ${
+    $d A f $.  $d B f $.
+    $( The set exponentiation ` ( B ^m A ) ` is a subset of the class of
```

Comment:
```suggestion
    $( The set ` ( B ^m A ) ` is included in the class of
```
(or "class", but as a function value it is a set, so as you want.)

"set exponentiation" is the operation, not its result.

Reply:
I would like to keepmthe term "set exponentiation" here (to distinguish it from the "class of functions" `{ f | f : A --> B }`), so I will replace  "set exponentiation" by "value of the set exponentiation".
Furthermore, I want to keep "is a subset" instead of "is included", because I think its more precise.

Reply:
Sounds heavy, but ok. Resolving.